Fix build with gcc14, comment code broken with gcc14 and gtk+2, it is a nop anyway.

diff -rup funcs.c.orig funcs.c
--- a/funcs.c
+++ b/funcs.c
@@ -23,6 +23,7 @@ Foundation, Inc., 59 Temple Place - Suit
 #include <stdlib.h>
 #include <string.h>
 #include <locale.h>
+#include <ctype.h>
 #include <gtk/gtk.h>
 
 #include "stack.h"
diff -rup process_input.c.orig process_input.c
--- a/process_input.c
+++ b/process_input.c
@@ -30,6 +30,7 @@ Foundation, Inc., 59 Temple Place - Suit
 #include "editor.h"
 #include "error.h"
 #include "number.h"
+#include "stack.h"
 #include <locale.h>
 
 
diff -rup main.c.orig main.c
--- a/main.c
+++ b/main.c
@@ -53,7 +53,7 @@ void usage(char *str){
 }
 
    GtkWidget *main_w;
-main(int argc, char *argv[])
+int main(int argc, char *argv[])
 {
    int n;
    int rows, cols;
@@ -174,13 +174,13 @@ main(int argc, char *argv[])
 
    /* set the font if told */
    if(btn_font != NULL){
-      default_style = gtk_widget_get_default_style();
-      if(NULL == (new_font = gdk_font_load(btn_font))){
-         fprintf(stderr, "Unable to load font %s.\n", btn_font);
-         exit(0);
-      }
-      new_style = gtk_style_copy(default_style);
-      new_style->font_desc = new_font;
+      //default_style = gtk_widget_get_default_style();
+      //if(NULL == (new_font = gdk_font_load(btn_font))){
+      //   fprintf(stderr, "Unable to load font %s.\n", btn_font);
+      //   exit(0);
+      //}
+      //new_style = gtk_style_copy(default_style);
+      //new_style->font_desc = new_font;
 /* BDD - No longer in Gtk2.x */
 /*      gtk_widget_set_default_style(new_style); */
    }
